weasyl/templates/help/community.html

```diff
@@ -22,20 +22,29 @@ <h4>A. Right to Submit</h4>
         <li><strong>Copyright and Trademark:</strong> The creator of the content of a submission holds the copyright and rights to its distribution; unless the content is trademarked by someone else, is considered harassment, or is otherwise a violation of Weasyl policy, the submission will not be removed. For further information, see our <a href="/policy/copyright">Copyright Policy</a>.</li>
       </ol>
 
-      <h4>B. Accurately Describing Content</h4>
+      <h4>B. Rating and Describing Your Content</h4>
+
+      <ul>
+        <li><strong>Terminology</strong>
+          <p>A “Submission” is any artwork, story, comic page, still image, or animation that the user uploads to the site. “Content” refers to actions, poses, points of view or mise-en-scene involving characters (see below) in the submission.</p>
+          <p>A “Character” is a drawn or described depiction of a conscious being (e.g., a furry, an anthropomorphized entity, an animal, a human, a robot, or any combination of these) in the submission. A “Cub” (sometimes called a “Babyfur” or “kidfur”) is any character with recognizably humanoid body proportions of children or infants (see below). We are using the term “cub” here as most characters on Weasyl are anthropomorphic animals. NOTE: Not to be confused with a “minor” (i.e., a user of the site who is under the real-life age of 18 and restricted to viewing General content).</p>
```

Comment:
This feels like a lot of explanation, when we could go with something like "Young character" in place of cub, which removes the need for one sentence of explanation (that we're using "Cub" because most characters on Weasyl are anthropomorphic).
It also neatly removes the contentious in-fandom terminology and disagreements vis-à-vis, babyfur/kidfur/cub. Specifically the "cub means cub porn" debate.